Controller.php

```diff
@@ -163,6 +186,17 @@ public function index($errorMessage = false)
                     ],
                 ],
             ],
+            'isConnectAccountsActivated' => $isConnectAccountsActivated,
+            'radioOptions' => [
+                'connectAccounts' => 'Quick connect with Google Analytics (recommended)',
```

Comment:
@snake14 This should be in a translation file

Reply:
Thank you @AltamashShaikh . I'll move those strings to a translation file.